pages/nav.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+
+export default function Nav() {
+  let Links = [
+    { name: "HOME", link: "/home" },
+    { name: "SERVICE", link: "/" },
+    { name: "ABOUT", link: "/" },
+    { name: "BLOG'S", link: "/" },
+    { name: "CONTACT", link: "/" },
+  ];
+  let [open, setOpen] = useState(false);
```

Comment:
Write Type Defination for useState
For this case it will be like useState<boolean>(false);

Reply:
useState<boolean>(false)